scripts/templates/migrations.template.js

```diff
@@ -9,7 +9,8 @@ const witnetRequests = require("../../migrations/witnet.requests")
 const WitnetPriceFeed = artifacts.require("#artifact")
 
 module.exports = async function (deployer, network, _accounts) {
-  network = network.split("-")[0]
+  let isFork = network.split("-")[1] === "fork"
```

Comment:
What's this for?

Reply:
distinguish when a truffle dry-run is going on

Reply:
Makes sense